### PR TITLE
allow the sns notifier to be disabled in config

### LINKED ISSUE #888: "missing region in config" errors when running content server tests on travis
Ref https://github.com/mozilla/fxa-content-server/issues/2187

An example from https://travis-ci.org/mozilla/fxa-content-server/builds/53189367

> 13:41:31.317Z ERROR fxa-auth-server: (op=Notifier.publish)
> err: {
>   "message": "Missing region in config",
>   "code": "ConfigError",
>   "time": "2015-03-05T13:41:31.317Z"
> }

I realize this is most likely an error in our configuration, but I'm posting it here to get the folks who know the most about the auth server involved.

### PR DESCRIPTION
**Pull request by dannycoates**
Fixes #888 

Not the prettiest, but this way doesn't require an config schema changes.

### CODE REVIEW

**Comment by dannycoates** [comment]
looking for r?

File: config/config.js [hunk 1]
```diff
@@ -182,7 +182,7 @@ module.exports = function (fs, path, url, convict) {
       default: 1
     },
     snsTopicArn: {
-      doc: 'Amazon SNS topic on which to send account event notifications',
+      doc: 'Amazon SNS topic on which to send account event notifications. Set to "disabled" to turn of the notifier',
```
Comment (rfk) [inline config/config.js]: s/of/off/

**Comment by rfk** [comment]
looks good, thanks @dannycoates

**Comment by shane-tomlinson** [comment]
Thanks for this @dannycoates!

**Pull request merged**